readme.md

```diff
@@ -46,19 +46,21 @@ git clone https://github.com/Laravel-Backpack/demo.git backpack-demo
 
 2) Set your database information in your .env file (use the .env.example as an example);
 
-3) Run in your backpack-demo folder:
+3) Set the APP_URL in .env, this value should match the URL you use to access your application in a browser.
```

Comment:
```suggestion
3) Make sure the `APP_URL` in your .env file is correctly pointing to the URL you use to access your application in the browser, for example: `http:127.0.0.1:8000` or `http://something.test`
```

Reply:
Yes! Having example is better 👍 

I used the exact line which Laravel Docs uses 😬
![Screenshot 2024-01-30 at 7 53 38 PM](https://github.com/Laravel-Backpack/demo/assets/8214221/d282fc7d-c3f2-4d66-b06d-b888aecdd97c)